pkg/server/update_runtime_config.go

```diff
@@ -17,14 +17,12 @@ limitations under the License.
 package server
 
 import (
-	"errors"
-
 	"golang.org/x/net/context"
 
 	runtime "k8s.io/kubernetes/pkg/kubelet/apis/cri/v1alpha1"
 )
 
 // UpdateRuntimeConfig updates the runtime config. Currently only handles podCIDR updates.
 func (c *criContainerdService) UpdateRuntimeConfig(ctx context.Context, r *runtime.UpdateRuntimeConfigRequest) (*runtime.UpdateRuntimeConfigResponse, error) {
-	return nil, errors.New("not implemented")
+	return &runtime.UpdateRuntimeConfigResponse{}, nil
```

Comment:
maybe a comment or todo to investigate if / when we need to update..  Returning no error here with no update if they passed in something to be updated could lead the user astray?